module_analysis_price/models/ir_module_type.py

```diff
@@ -42,8 +48,12 @@ def _compute_total_price(self, case):
             )
             record[f"{case}_year_price"] = record[f"{case}_monthly_price"] * 12
 
-    @api.depends("installed_module_ids.code_qty")
+    @api.depends("module_ids.code_qty")
```

Comment:
@PierrickBrun I have an issue the field module_ids does not exists ? Maybe you miss to commit something ?

Reply:
https://github.com/akretion/server-tools/commits/14.0-module_analysis_all_modules